File: ejercicios/python/ejercicio005.py
from math import *

x1 = float(input("ingrese la coordenada x1 del punto 1:? "))
y1 = float(input("ingrese la coordenada y1 del punto 1:? "))
x2 = float(input("ingrese la coordenada x2 del punto 2:? "))
y2 = float(input("ingrese la coordenada y2 del punto 2:? "))
d = sqrt((x2 - x1) ** 2 + (y2 - y1) ** 2 )
print("La distancia entre (" + str(x1) + "," + str(y1) + ") y (" + str(x2) + "," + str(y2) + ") es" + str(d))

File: ejercicios/c++/ejercicio005.cpp
#include<iostream>
#include<cmath>
using namespace std;

int main() {

	double d;
	double x1;
	double x2;
	double y1;
	double y2;

	cout<<"Ingrese la coordenada x1 del punto 1:? ";
	cin>>x1;
	cout<<"Ingrese la coordenada y1 del punto 1:? ";
	cin>>y1;
	cout<<"Ingrese la coordenada x2 del punto 2:? ";
	cin>>x2;
	cout<<"Ingrese la coordenada y2 del punto 2:? ";
	cin>>y2;
	d=sqrt((x2-x1)*(x2-x1)+(y2-y1)*(y2-y1));
	cout<<"La distancia entre ("<<x1<<","<<y1<<") y ("<<x2<<","<<y2<<") es "<<d<<endl;

	return 0;
}

